.vscode/launch.json

```
@@ -5,21 +5,26 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "name": "news_curation",
+      "name": "curatable",
       "request": "launch",
       "type": "dart"
     },
     {
-      "name": "news_curation (profile mode)",
+      "name": "backend",
+      "cwd": "backend",
       "request": "launch",
       "type": "dart",
-      "flutterMode": "profile"
+      "program": "bin/server.dart",
     },
     {
-      "name": "backend",
+      "name": "backend (emulator)",
       "cwd": "backend",
       "request": "launch",
-      "type": "dart"
+      "type": "dart",
+      "program": "bin/server.dart",
+      "args": [
+        "--dart-define", "BACKEND=EMULATOR"
```

I wanted to let you know you can write them this way as well
```suggestion
        "--dart-define=BACKEND=EMULATOR"
```

---

Cool! Maybe that's why it's not working?  I think I had it in my head that they needed to be separate arguments but that seems wrong now that you point it out... thanks!

---

The way you wrote it works as well you can write it both ways